heartbeat/docs/heartbeat-options.asciidoc

```diff
@@ -482,7 +482,8 @@ Under `check.request`, specify these options:
 
 *`method`*:: The HTTP method to use. Valid values are `"HEAD"`, `"GET"` and
 `"POST"`.
-*`headers`*:: A dictionary of additional HTTP headers to send.
+*`headers`*:: A dictionary of additional HTTP headers to send. By default heartbeat
```

Comment:
Nit: It would be nice to have a header example posted here. So if someone configures a tool where he needs to filter on the headers, he can just copy-paste it here (or at least have the pattern).